runelite-client/src/main/java/net/runelite/client/plugins/wiki/WikiPlugin.java

```diff
@@ -204,6 +220,17 @@ private void onWidgetHiddenChanged(WidgetHiddenChanged ev)
 		}
 	}
 
+	@Subscribe
+	public void onConfigChanged(ConfigChanged event)
+	{
+		if (event.getGroup().equals(CONFIG_GROUP_KEY))
+		{
+			// Since the Widget menu is configured on startup, it's easiest to just rebuild it when the config changes
+			shutDown();
+			startUp();
```

Comment:
You should probably use `addWidgets` instead of `startUp` and move the code in `shutDown` to a new method instead and call that here.

Reply:
How does this look now?